Makefile

```diff
@@ -208,6 +208,9 @@ GO_TEST = $(GOTESTSUM) $(GOTESTSUM_ARGS) --
 endif
 
 COUNT ?= 1
```

Comment:
What if instead we defaulted to unset, and if it's not empty, then set TEST_COUNT? Otherwise, you can't specify COUNT=1 and have it force running a test w/o hitting the cache.

Reply:
ACK - can drop this commit & do it stand-alone